lib/SessionManager.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Created by elad.benedict on 9/6/2015.
+ */
+
+var persistenceFormat = require('./PersistenceFormat');
+var glob = require('glob');
+var config = require('./Configuration');
+var Q = require('q');
+var qio = require('q-io/fs');
+var _ = require('underscore');
+
+module.exports = (function(){
+
+    var isNewSession = function isNewSession(entryId) {
+        return Q.fcall(function () {
+            var entryFolder = persistenceFormat.getEntryFullPath(entryId);
```

Comment:
this is a sync call that only concatenate strings. would you still like to wrap it as a promise?